cvat-core/src/server-proxy.js

```diff
@@ -619,19 +624,91 @@
                     throw generateError(errorData);
                 }
 
-                onUpdate('The data are being uploaded to the server..');
-                try {
-                    await Axios.post(`${backendAPI}/tasks/${response.data.id}/data`, taskData, {
-                        proxy: config.proxy,
-                    });
-                } catch (errorData) {
-                    try {
-                        await deleteTask(response.data.id);
-                    } catch (_) {
-                        // ignore
+                onUpdate('The data are being uploaded to the server 0%');
+
+                async function chunkUpload(file) {
+                    const chunkSize = 1024 * 1024 * 100; // 100 mb
+                    const totalChunks = Math.ceil(file.size / chunkSize);
+                    let currentChunkNumber = 1;
+                    async function upload() {
+                        const prevChunkNumber = currentChunkNumber - 1;
+                        const fileChunk = file.slice(prevChunkNumber * chunkSize, Math.min(prevChunkNumber * chunkSize + chunkSize, file.size));
+                        taskData.set('client_files[0]', fileChunk);
+                        taskData.set('chunk_file_name', file.name);
+                        taskData.set('end_of_upload', currentChunkNumber >= totalChunks);
+                        taskData.set('use_chunk_upload', true);
```

Comment:
According to current design I think we should use query parameters instead of body arguments. 
?action=append (on the server can we understand what data client sent (chunk of a single file, or one file? If yes, we do not need different flags to chunks and bulks))
?action=submit (end of upload sent in a different request with no data)

Reply:
Done